From 976f28f7be80c1ea9f09b74a6213f612ab015918 Mon Sep 17 00:00:00 2001
From: mcpierce <mcpierce@13f79535-47bb-0310-9956-ffa450edef68>
Date: Mon, 12 Nov 2012 16:38:14 +0000
Subject: [PATCH] Changed systemd support to requiring Networking start first.

Before the qpidd service was started in parallel to networking. Now it's
started after networking is started.

git-svn-id: https://svn.apache.org/repos/asf/qpid/trunk@1408346 13f79535-47bb-0310-9956-ffa450edef68
---
 qpid/cpp/etc/qpidd.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qpid/cpp/etc/qpidd.service.in b/qpid/cpp/etc/qpidd.service.in
index ec89e12..a654983 100644
--- a/qpid/cpp/etc/qpidd.service.in
+++ b/qpid/cpp/etc/qpidd.service.in
@@ -1,7 +1,7 @@
 [Unit]
 Description=An AMQP message broker daemon.
 Documentation=man:qpidd(1) http://qpid.apache.org/
-After=network.target
+Requires=network.target
 
 [Service]
 User=qpidd
-- 
1.7.11.7

